charts/sn-platform/templates/proxy/proxy-cluster.yaml

```diff
@@ -121,6 +123,16 @@ spec:
     imagePullSecrets:
 {{ toYaml .Values.global.imagePullSecrets | indent 6 }}
     {{- end }}
+  {{- if .Values.istio.enabled }}
```

Comment:
it doesn't seems we can configure gateway in PulsarProxy: https://github.com/streamnative/pulsar-operators/blob/master/pulsar-operator/api/v1alpha1/pulsarproxy_types.go#L85-L95

Reply:
That's still in develop, I'll draft this.